token/cli/tests/command.rs

```diff
@@ -3842,4 +3842,27 @@ async fn group(test_validator: &TestValidator, payer: &Keypair) {
         extension_pointer.group_address,
         Some(mint).try_into().unwrap()
     );
+
+    let new_max_size = 12;
+
+    // Update token-group max-size
+    process_test_command(
+        &config,
+        payer,
+        &[
+            "spl-token",
+            CommandName::UpdateGroupMaxSize.into(),
+            &mint.to_string(),
+            &new_max_size.to_string(),
+        ],
+    )
+    .await
+    .unwrap();
+
+    let account = config.rpc_client.get_account(&mint).await.unwrap();
+
+    let updated_mint_state = StateWithExtensionsOwned::<Mint>::unpack(account.data).unwrap();
+
+    let updated_extension = updated_mint_state.get_extension::<TokenGroup>().unwrap();
+    assert_eq!(updated_extension.max_size, new_max_size.into());
```

Comment:
Just a side note: you can't do this yet until we have the other instructions, but it would be good to be able to verify the CLI response message if you try to update the max size to a value below the number of current members. Just something to keep in mind when that goes in later!

Reply:
I agree, I can loop back to this when I am buttoning it all up.